client/driver/lxc.go

```diff
@@ -153,18 +153,20 @@ func (d *LxcDriver) Validate(config map[string]interface{}) error {
 		return err
 	}
 
-	volumes, _ := fd.GetOk("volumes")
-	for _, volDesc := range volumes.([]interface{}) {
-		volStr := volDesc.(string)
-		paths := strings.Split(volStr, ":")
-		if len(paths) != 2 {
-			return fmt.Errorf("invalid volume bind mount entry: '%s'", volStr)
-		}
-		if len(paths[0]) == 0 || len(paths[1]) == 0 {
-			return fmt.Errorf("invalid volume bind mount entry: '%s'", volStr)
-		}
-		if paths[1][0] == '/' {
-			return fmt.Errorf("unsupported absolute container mount point: '%s'", paths[1])
+	volumes, ok := fd.GetOk("volumes")
+	if ok {
```

Comment:
Might be cleaner to `if !ok` and return early here?

Reply:
I'm hesitant to return early in nomad code base - we have many long functions where more code gets appended or shuffled, and early returns for non error can cause unexpected behavior.  I'd rather extract functions, but given that this is basically dead code at this point, I'd rather leave as-is.